### DISCUSSION TITLE
Location - PLEASE SAVE US ALL! your our only hope.

### DISCUSSION THREAD

**Discussion by Alzibub** [jianastrero/hogwarts-legacy-save-editor#8] [Ideas]
I have found your save game editor after finding my save game is stuck falling forever.. I have no other game to restore too

I tried looking through the sav file and tried using starting location from a new save game.. but wouldn’t load.

There is a ton of us on Twitter that need this fix. If you could code in an ability to restore location to hogwarts.
You would be our saviour!

**Comment by jianastrero** [comment]
Could you attach your game saves?

No promises, but, i will try my best.

**Comment by Alzibub** [comment]
Thank you so much, find the save file attached.

[Drop link to hogwarts legacy save file ](https://www.dropbox.com/s/hn3leac3hhea5p2/HL-01-00.sav?dl=0)
